NOTICE.txt

```diff
@@ -7,6 +7,20 @@ NOTICES:
 This document includes a list of open source components used in Mattermost Web App, including those that have been modified.
 
 --------
+
+## @types/enzyme
```

Comment:
This commit has many ``@types``: https://github.com/mattermost/mattermost-webapp/commit/09d9fc3a00b176831e4cf3c3ceef4d25bde4dc14#diff-b9cfc7f2cdf78a7f4b91a753d10865a2. Is there a general one for all of them?

Reply:
Looks like they come from here: https://github.com/DefinitelyTyped/DefinitelyTyped